src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -539,11 +608,11 @@ private static IServiceProvider CreateServiceProvider(IVsMonitorSelection vsMoni
         private static void CheckConnectedStatusIsChecked(Mock<ISonarQubeService> serviceMock) =>
             serviceMock.Verify(x => x.GetServerInfo(), Times.Once);
 
-        private static void CheckIssuesAreFetched(Mock<ISonarQubeService> serviceMock) =>
-            serviceMock.Verify(x => x.GetTaintVulnerabilitiesAsync(SharedProjectKey, null, It.IsAny<CancellationToken>()), Times.Once);
+        private static void CheckIssuesAreFetched(Mock<ISonarQubeService> serviceMock, string projectKey, string branch) =>
```

Comment:
Caller now has to be explicitly about which project/branch values they expected to be used, instead of making assumptions.